support-frontend/assets/helpers/deliveryCheck.js

```diff
@@ -1,57 +1,8 @@
-  'TW11', 'TW12', 'TW13', 'TW14', 'TW105', 'TW106', 'TW107', 'TW110', 'TW118', 'TW119', 'TW121', 'TW122', 'TW123', 'TW134', 'TW135', 'TW136', 'TW137', 'TW140', 'TW148', 'TW149', 'TW151', 'TW152', 'TW153', 'TW165', 'TW166', 'TW167', 'TW170', 'TW178', 'TW179', 'TW181', 'TW182', 'TW183', 'TW184', 'TW197', 'TW25', 'TW26', 'TW27', 'TW200', 'TW208', 'TW31', 'TW32', 'TW33', 'TW34', 'TW39', 'TW45', 'TW46', 'TW47', 'TW50', 'TW59', 'TW74', 'TW75', 'TW76', 'TW77', 'TW80', 'TW88', 'TW89', 'TW91', 'TW92', 'TW93', 'TW94',
-  'UB11', 'UB12', 'UB13', 'UB100', 'UB108', 'UB109', 'UB111', 'UB24', 'UB25', 'UB32', 'UB34', 'UB35', 'UB40', 'UB48', 'UB49', 'UB54', 'UB55', 'UB60', 'UB68', 'UB69', 'UB77', 'UB79', 'UB81', 'UB82', 'UB83', 'UB94', 'UB95', 'UB96',
-  'W104', 'W105', 'W106', 'W111', 'W112', 'W113', 'W114', 'W120', 'W127', 'W128', 'W129', 'W130', 'W138', 'W139', 'W13O', 'W140', 'W148', 'W149', 'W1A3', 'W1B1', 'W1B3', 'W1B4', 'W1C2', 'W1D3', 'W1D4', 'W1F0', 'W1F7', 'W1F8', 'W1F9', 'W1G0', 'W1G6', 'W1G7', 'W1G8', 'W1G9', 'W1H1', 'W1H2', 'W1H5', 'W1H6', 'W1H7', 'W1J0', 'W1J5', 'W1J6', 'W1J7', 'W1J8', 'W1K1', 'W1K2', 'W1K3', 'W1K4', 'W1K5', 'W1K6', 'W1K7', 'W1S1', 'W1S2', 'W1S3', 'W1S4', 'W1T1', 'W1T2', 'W1T4', 'W1T6', 'W1T7', 'W1U1', 'W1U2', 'W1U3', 'W1U4', 'W1U5', 'W1U6', 'W1U7', 'W1U8', 'W1W5', 'W1W6', 'W1W7', 'W1W8', 'W21', 'W22', 'W23', 'W24', 'W25', 'W26', 'W30', 'W36', 'W37', 'W38', 'W39', 'W41', 'W42', 'W43', 'W44', 'W45', 'W51', 'W52', 'W53', 'W54', 'W55', 'W60', 'W67', 'W68', 'W69', 'W71', 'W72', 'W73', 'W84', 'W85', 'W86', 'W87', 'W91', 'W92', 'W93', 'WC1B3', 'WC1H0', 'WC1H9', 'WC1N1', 'WC1N2', 'WC1N3', 'WC1R5', 'WC1V6', 'WC1V7', 'WC1X0', 'WC1X8', 'WC1X9', 'WC2A1', 'WC2A3', 'WC2E7', 'WC2E8', 'WC2E9', 'WC2H9', 'WC2N4', 'WC2N5', 'WC2N6', 'WC2R0', 'WC2R2', 'WC2R3',
-  'WD172', 'WD173', 'WD174', 'WD180', 'WD187', 'WD188', 'WD194', 'WD195', 'WD196', 'WD197', 'WD231', 'WD232', 'WD233', 'WD234', 'WD244', 'WD245', 'WD246', 'WD247', 'WD250', 'WD257', 'WD258', 'WD259', 'WD31', 'WD33', 'WD34', 'WD35', 'WD36', 'WD37', 'WD38', 'WD48', 'WD50', 'WD61', 'WD62', 'WD63', 'WD64', 'WD65', 'WD77', 'WD78', 'WD79'];
-
-const M25_POSTCODE_PREFIXES = M25_POSTCODE_OLD_PREFIXES.concat(M25_NEW_PREFIXES);
+const M25_POSTCODE_PREFIXES = getGlobal('homeDeliveryPostcodes') || [];
```

Comment:
wasn't sure what to do if the value does not exist as the type checker will not notice any more - as it is it will block all purchases!
The post deploy tests seem to check subs card rather than home delivery at the moment. https://github.com/guardian/support-frontend/blob/2df3358d1e443d21c2c6b82db256631937268c0e/support-frontend/test/selenium/subscriptions/CheckoutsSpec.scala#L48-L52

Reply:
I think this is low risk because it's rendered directly into the page. We could add another test for HD or change the existing one if you're worried?

Reply:
There's no test to ensure that it really is rendered into the page.  But I agree that part is unlikely to be broken inadvertently.  I will leave it as it is.